test/persistence-hooks.suite.js

```diff
@@ -86,6 +86,18 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           });
       });
 
+      it('should trigger access hook once, if near filter is used', function(done) {
+        monitorHookExecution();
+        TestModel.find(
+          { where:
+            { location: { near: '10,20', maxDistance: '10', unit: 'meters' }},
+          }, function(err, list) {
+          if (err) return done(err);
+          hookMonitor.names.should.eql(['access']);
```

Comment:
Is there any GH issue to keep track of this bug? Are you planning to fix this as part of  strongloop/loopback-connector-mongodb#285? Either way, please create a checklist item or a new GH issue so that we don't forget about this problem.